css/src/metabox.css

```diff
@@ -1,6 +1,7 @@
 @import url(../src/icons.css);
 @import url(../src/modal.css);
 @import "../../node_modules/draft-js-mention-plugin/lib/plugin.css";
```

Comment:
Isn't this replaced then?

Reply:
Good catch. Also found `css/src/search-appearance.css` had the same import